jsk_spot_robot/jsk_spot_startup/launch/jsk_spot_bringup.launch

```diff
@@ -30,5 +32,12 @@
   <!-- teleop node -->
   <include
       file="$(find jsk_spot_teleop)/launch/teleop.launch"
-      />
+      >
+      <arg name="pad_type" value="R1" />
+  </include>
+  <include
```

Comment:
Do we need to use `<if>` and `<unless>` for these <include> blocks? It seems like that the current code will simultaneously create two teleop.launch.

Reply:
that is intended behavior. I think both pads have pros. and cons. But if it is required to stop and launch ’jsk_spot_bringup.launch’ when switching a controller, it is inconvenient. So this PR uses both controllers simultaneously.

Maybe it depends on use cases. Please tell me if there is any suggestion.